README.md

```diff
@@ -1,91 +1,97 @@
-
-<!---
-
-This README is automatically generated from the comments in these files:
-app-localize-behavior.html
-
-Edit those files, and our readme bot will duplicate them over here!
-Edit this file, and the bot will squash your changes :)
-
-The bot does some handling of markdown. Please file a bug if it does the wrong
-thing! https://github.com/PolymerLabs/tedium/issues
-
--->
-
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/app-localize-behavior.svg)](https://www.npmjs.com/package/@polymer/app-localize-behavior)
 [![Build status](https://travis-ci.org/PolymerElements/app-localize-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/app-localize-behavior)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/app-localize-behavior)
 
-
-## Polymer.AppLocalizeBehavior
-
-`Polymer.AppLocalizeBehavior` wraps the [format.js](http://formatjs.io/) library to
+## &lt;app-localize-behavior&gt;
+`app-localize-behavior` is a behavior that wraps the [format.js](http://formatjs.io/) library to
 help you internationalize your application. Note that if you're on a browser that
 does not natively support the [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl)
 object, you must load the polyfill yourself. An example polyfill can
 be found [here](https://github.com/andyearnshaw/Intl.js/).
 
-`Polymer.AppLocalizeBehavior` supports the same [message-syntax](http://formatjs.io/guides/message-syntax/)
-of format.js, in its entirety; use the library docs as reference for the
-available message formats and options.
+See: [Documentation](https://www.webcomponents.org/element/@polymer/app-localize-behavior),
+  [Demo](https://www.webcomponents.org/element/@polymer/app-localize-behavior/demo/demo/index.html).
 
-Sample application loading resources from an external file:
-
-```html
-<dom-module id="x-app">
-   <template>
-    <div>{{localize('hello', 'name', 'Batman')}}</div>
-   </template>
-   <script>
-      Polymer({
-        is: "x-app",
+## Usage
 
-        behaviors: [
-          Polymer.AppLocalizeBehavior
-        ],
-
-        properties: {
-          language: {
-            value: 'en'
-          },
-        }
-
-        attached: function() {
-          this.loadResources(this.resolveUrl('locales.json'));
-        },
-      });
-   &lt;/script>
-</dom-module>
+### Installation
+```
+npm install --save @polymer/app-localize-behavior
 ```
 
-Alternatively, you can also inline your resources inside the app itself:
-
+### In an html file using the localized element
 ```html
-<dom-module id="x-app">
-   <template>
-    <div>{{localize('hello', 'name', 'Batman')}}</div>
-   </template>
-   <script>
-      Polymer({
-        is: "x-app",
-
-        behaviors: [
-          Polymer.AppLocalizeBehavior
-        ],
+<html>
+  <head>
+    <!-- You MUST import this library first for the behaviour to work -->
```

Comment:
IMHO, we should use the 'behavior' spelling in the docs, given that it's spelled that way in the name of the package and export. (There's a couple of other places in this PR too.)

Reply:
ughhhhhhhhhhhhhhhhhh

Reply:
Sorry? 🇨🇦

Reply:
(i'm not doing this. you'll be delighted to find i've been peppering our code base with extraneous 'u' spelling for yearssssss)